### PR TITLE
ENH Keras Use table instead of dictionary for hyperparameters in model card

### PR DESCRIPTION
**Pull request by merveenoyan**
This PR turns hyperparameters dictionary to table on model card.
Before: https://huggingface.co/merve/model-card-example
After: https://huggingface.co/merve/hyperparam_table
Ekran Resmi 2022-05-17 12 01 27
Ekran Resmi 2022-05-17 12 01 49
Ekran Resmi 2022-05-17 12 02 28
(You can scroll for more hyperparams)

### CODE REVIEW

File: src/huggingface_hub/keras_mixin.py [hunk 1]
```diff
@@ -27,16 +27,28 @@
     import tensorflow as tf
 
 
-def _extract_hyperparameters_from_keras(model):
+def _create_hyperparameter_table(model):
+    """This function parses hyperparameter dictionary into a markdown table."""
```
Comment (osanseviero) [inline src/huggingface_hub/keras_mixin.py:31]:
```suggestion
    """Parse hyperparameter dictionary into a markdown table."""
```

**Review by osanseviero — APPROVED** [review]
Looks nice!

**Comment by merveenoyan** [comment]
thank you🥹

**Comment by merveenoyan** [comment]
@osanseviero you can merge as you wish :)

**Review by osanseviero — APPROVED** [review]
Looks good! Thanks for the PR

**Review by adrinjalali — COMMENTED** [review]
I would have liked this to be merged with tests and a bit more documentation.

**Comment by osanseviero** [comment]
This is not a functional change, i.e. this is not adding new files, saving new things, etc. This is just doing a markdown/style change, which I don't think we should have tests for all the parts of the markdown. It would make doing any style change slower and be a bit of unnecessary work at this stage IMO. 

cc @LysandreJik as well if he disagrees on it

**Pull request merged**